SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTest.cpp

```diff
@@ -36,6 +36,27 @@ constexpr float FINISH_TEST_GRACE_PERIOD_DURATION = 2.0f;
 ASpatialFunctionalTest::ASpatialFunctionalTest()
 	: Super()
 	, FlowControllerSpawner(this, ASpatialFunctionalTestFlowController::StaticClass())
+	, bIsStandaloneTest(false)
+{
+	Init();
+}
+
+ASpatialFunctionalTest::ASpatialFunctionalTest(const EMapCategory MapCiCategory, const int32 NumberOfClients /*=1*/,
+											   const FVector& InTestPositionInWorld /*=FVector::ZeroVector*/)
+	: Super()
+	, FlowControllerSpawner(this, ASpatialFunctionalTestFlowController::StaticClass())
+	, bIsStandaloneTest(true)
+	, TestPositionInWorld(InTestPositionInWorld)
+{
+	Init();
```

Comment:
You can just call the other constructor in your initializer list, no need to split this out in a Init() function.